app/assets/javascripts/student_profile/provided_by_educator_dropdown.js

```diff
@@ -58,12 +61,13 @@
           self.props.onUserDropdownSelect(ui.item.value);
         }
       });
+
+      $('body').bind('click.closeMenu', this.closeMenu);
```

Comment:
I agree, I thought about implementing something similar to [this]( https://github.com/facebook/react/issues/579), but for simplicity I thought this was ideal.

I could do
``` javascript
  componentDidMount () {
    window.__myapp_container.addEventListener('click', this.closeMenu)
  }

  componentWillUnmount () {
    window.__myapp_container.removeEventListener('click', this.closeMenu)
  }
```

to be more idiomatic react.